index.md

```diff
@@ -83,3 +83,4 @@ The [GitHub Pages version](https://karpet.github.io/gov-tech-reading-list/) of t
 * [Reading list on digital gov](https://docs.google.com/document/d/1xuTuEPjMeiIv1Kwxv5MYXFAv1lBnLdO15YuL_oKYxSI/edit) via OpenEugene
 * [Crowdlaw for Congress](https://congress.crowd.law/)
 * [Decolonizing Civic Tech](https://decolonizingcivictech.com/)
+* [Jaimie Boyd's Digital government reading list](https://docs.google.com/document/d/1xuTuEPjMeiIv1Kwxv5MYXFAv1lBnLdO15YuL_oKYxSI/edit
```

Comment:
if you look at line 83 above this one, it's the same google doc link.

great stuff though!

Reply:
Drat! Sorry about that.